playbooks/tasks/portal-accounts-generate-jwks.json.yml

```diff
@@ -0,0 +1,47 @@
+---
+# Generate Accounts jwks.json config file according to skynet-accounts README.
+# Sometimes the generated jwks.json file is empty so we retry until we
+# successfully generate the content.
+
+- name: Increment jwks retry count
+  set_fact:
+    jwks_retries_count: "{{ jwks_retries_count | int + 1 }}"
+
+- name: Generate JWKS config according to README in skynet-accounts repo
+  local_action:
```

Comment:
is this ran on host machine or the deployment machine?

Reply:
We have to upload it to LastPass so it is run on deployment machine.